sensor_msgs/Image heightfield
int32 n_rows
int32 n_cols
float32 resolution_x
float32 resolution_y
float32 resolution_z